process-document/src/main/java/com/ritense/processdocument/web/rest/ProcessDocumentResource.java

```diff
@@ -128,11 +128,11 @@ public ResponseEntity<ProcessDocumentDefinition> getProcessDocumentDefinition(
     }
 
     @GetMapping("/v1/process-document/instance/document/{documentId}")
-    public ResponseEntity<List<? extends ProcessDocumentInstance>> findProcessDocumentInstances(
+    public ResponseEntity<List<ProcessDocumentInstanceDto>> findProcessDocumentInstances(
```

Comment:
Isn't this a breaking change? Maybe we should create a v2

Reply:
Changed it. But v2 isn't needed